lessons/systems-administration/sysadmin-linux-navigation.md

```diff
@@ -1 +1,106 @@
+
+You can see that when we type run `pwd` before and after, our current working directory changed from `/home/hackabit` to `/tmp`.  You could also verify this by using `env` like we showed you earlier, feel free to try that if you like.
+
+## Section 3
+There are a few basic rules and caveats to keep in mind when trying to `cd` to a different directory.  One of these is to know the diffence between `absolute path` and `relative path`.
+
+`absolute path` is the full path to the file or directory you are aiming for.  These will always start from the root folder of the Linux filesystem, which is simply `/`.  From there, you'll need to type the name of every folder, followed by a `/` until you reach the directory that you're trying to get to.  `absolute path` is often preferred by some programs because it removes as much ambiguity as possible.
+
+`relative path` takes a little more explaining.  First we'll need to learn some "shortcuts" that are used to prevent having to type full, absolute paths every time you change directories.  These shortcuts are `.` which refers to your current directory (the same thing you would get from running `pwd`) and `..` which refers to the directory up one level from your current directory.  This is easier to see from an example...  
+  
+![relativePath Demo](.rsrc/relativePath.png)
+
+You'll see that with a relative path, we don't actually have to type the full pathname all the way from the root folder, instead we can change directories *relative* to or current directory.  
+  
+One more caveat with relative paths is that you don't always have to type the single `.` to change directories if the target directory is within your current directory.  You can simply omit the `.` for convienience, but it's important to know that the `.` represents your current directory, because it comes into play in many other ways.
+
+## Section 4
+Now that we know how to get our bearings and move around, we need to be able to look at different files and folders contained in these directories.  For that, we'll introduce a tool called `ls` which is short for "list."
+
+`ls` works by simply listing all the files and folders in your current working directory by default.  But you can also give it an `absolute path` or `relative path` to list things that are *not* in your current working directory.  An example:  
+  
+![ls Demo](.rsrc/ls.png)
+
+One thing to notice here is that `ls` did not change our current directory, even when we used it to look inside other folders.  We can also see that there are many different ways to use `ls`: by itself, with an `absolute path`, or with a `relative path`.    
+  
+But `ls` is much more powerful and flexible than that!  It has additional `flags`, or options that can be applied to it, that list even more detail.  One of these that is very useful is `ls -l`, which lists things in "long" format that shows permissions, file owner, group owner, file sizes, and last modified times.  
+  
```

Comment:
can we also add in the -a flag to talk about hidden files? It would also give the chance to re-emphasize the ideas of current directory and up one directory.